RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release Notes
 
+## in develop
+
+* `dxda` update to `v0.5.10` which adds a retry for file download 
```

Comment:
Nit: I'd only maybe make it even more clear which error it fixes so the client knows immediately it was "their" issue, e.g. which adds a retry for file download fixing the "context canceled" error (or similar)

Reply:
Fixed with more info assuming "their" means "originated form dxda".